docs/01-Get Started/visual-testing.mdx

```diff
@@ -0,0 +1,148 @@
+
+## Screenshot a website
+
+The first step to visual testing is to screenshot your website or application. Up to you to test the components, the full pages or specific steps of your checkout tunnel.
+
+> Tip: It is easier to visually check page content than to assert each element's text. Moreover, there is no code maintenance to update visual test.
+
+We recommend to lean on an end-to-end testing frameworks to take the screenshots. They facilitate the navigation through website and generally allows to take screenshots in various browsers and screen resolutions. We provide integrations for some of them :
+
+<CiCards>
+  <CICardLink href="/cypress">
+    <CiCard>
+      <CiBody>
+        <CiCardImage>
+          <img src={CypressLogo} />
+        </CiCardImage>
+        <CiCardTitle>Cypress</CiCardTitle>
+      </CiBody>
+      <CiArrow />
+    </CiCard>
+  </CICardLink>
+  <CICardLink href="/puppeteer">
+    <CiCard>
+      <CiBody>
+        <CiCardImage>
+          <img src={PuppeteerLogo} style={{ maxHeight: 25, margin: "auto" }} />
+        </CiCardImage>
+        <CiCardTitle>Puppeteer</CiCardTitle>
+      </CiBody>
+      <CiArrow />
+    </CiCard>
+  </CICardLink>
+  <CICardLink href="/playwright">
+    <CiCard>
+      <CiBody>
+        <CiCardImage>
+          <img src={PlaywrightLogo} />
+        </CiCardImage>
+        <CiCardTitle>Playwright</CiCardTitle>
+      </CiBody>
+      <CiArrow />
+    </CiCard>
+  </CICardLink>
+</CiCards>
+
+This integration provide useful utilities to hide instable elements from screenshots. Read the dedicated section to learn more about screenshot stability.
+
+## Visual changes detection
+
+Once the [installation](/installation) is done, Argos will receive your application's screenshots when you create a pull request or merge a commit to the repository main's branch.
+
+Argos compare the screenshots with their baseline counterpart. If a visual change is detected, a red check status is added to the GitHub pull request.
+
+<img
+  src={githubCheckStatus}
+  alt="GitHub check status"
+  style={{ borderRadius: 8 }}
+/>
+
+Click on `Details` link to review the differences.
+
+## Review the visual changes
+
+The visual changes review happens on Argos.
+
+<img src={siriusBuild} alt="Example build" style={{ borderRadius: 8 }} />
+
+Navigate through the screenshot diffs with the thumbnail list, the arrow buttons or with the keyboard shortcuts.
+
+> Tip: Use the `(?)` shortcut to show the keyboard shortcuts list.
+
+The side-by-side UI is designed to easily spot visual changes and verify that screenshots match their expected results.
+
+#### Reject a build
+
+If you spot a visual bug or if a screenshot doesn't satisfy your expectations, click on `Review changes` button and select `Reject changes`. It will update the build status to `Changes rejected` and notify Github that the test is a failure.
```

Comment:
```suggestion
If you spot a visual bug or if a screenshot doesn't satisfy your expectations, click on "Review changes" button and select "Reject changes". It will update the build status to "Changes rejected" and notify GitHub that changes have been rejected.
```

Backquote are to highlight code, this is not code, use quotes.